script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -26,12 +29,12 @@ They're cheap, but cheap is better than wet. I'll run a bath, so if you get in a
 [sha:5b5d9592684a4d675a885f2801c80ae2a1d0c304]{
 -- Page 193, Offset 5343.
 -- 先輩はてきぱきと準備をしていく。
-Senpai readies everything quickly.
+Senpai briskly sets to work getting everything in order.
```

Comment:
"briskly sets to work" is an awkward expression.
> Senpai quickly gets to work getting everything in order.

Reply:
I was going to plead British to this one but yeah it really doesn't seem to be an expression. Still makes sense to me but will change.